### ISSUE TITLE
p2g and g2g environment 

### ISSUE THREAD

**Issue by songhan89** [ssec/polar2grid#161]
Hi, 

Just to report that I noticed if existing p2g env is already set, there is some conflict when I attempt to exec `geo2grid.sh`

This is not really an issue as it can be easily resolved, but thought would be good to let you know. Thanks.

Great work on g2g package by the way! I'm testing it out.

```

[songhan@h8-dps2 ~]$ /home/songhan/geo2grid/bin/geo2grid.sh 
/home/songhan/geo2grid/bin/geo2grid.sh: line 36: /home/songhan/polar2grid_v_2_0/bin/env.sh: No such file or directory
/home/songhan/geo2grid/bin/geo2grid.sh: line 45: python3: command not found

[songhan@h8-dps2 ~]$ export POLAR2GRID_HOME=/home/songhan/geo2grid

[songhan@h8-dps2 ~]$ /home/songhan/geo2grid/bin/geo2grid.sh 
/home/songhan/geo2grid/bin/geo2grid.sh: line 45: python3: command not found

[songhan@h8-dps2 ~]$ /home/songhan/geo2grid/bin/geo2grid.sh 
/home/songhan/geo2grid/bin/geo2grid.sh: line 45: python3: command not found

[songhan@h8-dps2 ~]$ export POLAR2GRID_HOME=/home/songhan/geo2grid

[songhan@h8-dps2 ~]$ /home/songhan/geo2grid/bin/geo2grid.sh 
/home/songhan/geo2grid/bin/geo2grid.sh: line 45: python3: command not found

[songhan@h8-dps2 ~]$ export PATH=$PATH:$GEO2GRID_HOME/bin

[songhan@h8-dps2 ~]$ /home/songhan/geo2grid/bin/geo2grid.sh 
usage: 
    geo2grid.sh -h
see available products:
    geo2grid.sh -r <reader> -w <writer> --list-products -f file1 [file2 ...]
basic processing:
    geo2grid.sh -r <reader> -w <writer> [options] -f file1 [file2 ...]
basic processing with limited products:
    geo2grid.sh -r <reader> -w <writer> [options] -p prod1 prod2 -f file1 [file2 ...]

```

**Comment by djhoese** [comment]
This is a known problem. There isn't really a way around that right now except to not combine the two together. FYI it is possible to call the `geo2grid.sh` and `polar2grid.sh` scripts directly without having to source the `env.sh` scripts first.

**Comment by djhoese** [comment]
In modern Polar2Grid (upcoming 3.0) and Geo2Grid (1.1), the POLAR2GRID_HOME and GEO2GRID_HOME variables should not be used internally and should have no effect if the user sets them before hand. Additionally, the `bin/` directory in the tarballs contain a minimal number of scripts or softlinks to common utilities. Both projects will depend on similar versions of python and the same python libraries and will be running all the same code so there should be minimal issues. That said, both project's tarballs currently include a `polar2grid.sh` and `geo2grid.sh` scripts so if you add both of them to your PATH there will be conflicts.

So bottom line is that this isn't supported and not expected to work, but in modern versions it will break **less** badly.
